api/src/main/java/org/openmrs/module/oauth2/api/model/LaunchValue.java

```diff
@@ -0,0 +1,78 @@
+package org.openmrs.module.oauth2.api.model;
```

Comment:
Let's move the SMART models to oauth2.api.mode.smart ? And I mean both the models (OA-9 and OA-10). Make this change and update the PR with a combined squashed commit (OA-9 and OA-10) 😄

Reply:
OK 👍